### PR TITLE
Integrate cookie-policy v3.0.4

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Add global-nav and cookie-policy dependencies
- Built with webpack

## QA
- Load the site
- Check the global nav works as expecred
- Check the cookie policy is there and works
- Go to the footer and manage your cookies and see it opens again

## Issue / Card
Fixes https://github.com/canonical-web-and-design/docs.ubuntu.com/issues/268

### CODE REVIEW

**Comment by pmahnke** [comment]
@anthonydillon for whatever reason I can't run it locally and the demo can't find the new js, I think the `Dockerfile` needs to also build the javascript as well.

**Comment by anthonydillon** [comment]
Ah probably, I will take a look

**Comment by anthonydillon** [comment]
@pmahnke done.

File: webpack.config.entry.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  "global-nav": "./static/js/global-nav.js",
```
Comment (pmahnke) [inline webpack.config.entry.js:2]: this still doesn't work on the demo... perhaps it should be like

```
  ["global-nav"]: "./static/js/global-nav.js",
  ["cookie-policy"]: "./static/js/cookie-policy.js",
```

**Review by pmahnke — CHANGES_REQUESTED** [review]
still not running on demos.haus

**Comment by anthonydillon** [comment]
@pmahnke sorry forgot to actually copy the JS to the dockerfile after building it.

**Comment by pmahnke** [comment]
@anthonydillon the build is failing - `rpc error: code = Unknown desc = executor failed running [/bin/sh -c yarn run build-js]: exit code: 1` I think the it should be static/js/dist ?

**Review by pmahnke — APPROVED** [review]
works!

**Comment by anthonydillon** [comment]
Phew, thanks.

**Pull request merged**